.github/workflows/pr-builder.yml

```diff
@@ -1,73 +1,42 @@
-name: pr-builder
+# This workflow will build the project on pull requests with tests
+# Uses:
+#   OS: ubuntu-lates
+#   JDK: Adopt JDK 8
+
+name: PR Builder
 
 on:
-  pull_request_target:
-    types: [labeled]
-    branches:
-      - main
-      - master
+  pull_request:
```

Comment:
I think this might be the more suitable one https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target since we want to build the forked repo

Reply:
Will have to add triggeres on `opened` and `edited` i guess